src/workspace-data/data-table/entity-service/AttributeInput.test.ts

```diff
@@ -14,9 +14,12 @@ describe('AttributeTypeInput', () => {
     );
 
     const radioButtons = getAllByRole('radio');
-    const labels = Array.from(radioButtons).map((el) => el.labels[0].textContent);
-
     expect(radioButtons.length).toBe(4);
+
+    const labels = Array.from(radioButtons).map((el) => {
+      const inputElement = el as HTMLInputElement;
+      return inputElement.labels ? inputElement.labels[0].textContent : '';
+    });
     expect(labels).toEqual(['String', 'Reference', 'Number', 'Boolean']);
   });
```

Comment:
please add AAA labels to these tests to help readability of each test's block sections:

// Arrange

// Act

// Assert